tests/OpenGraphParser/OpenGraphParserTest.php

```diff
@@ -321,6 +321,22 @@ public function testParseListOnlyReturnsResultsForEntriesWithoutExceptions() {
         $results = $this->subject->parseList(array('first','second','third'));
         $this->assertEquals(2, count($results));
     }
+    
+    public function testSettingNewCacheAdapterItAlsoShouldSetItOnTheFetchStrategy()
+    {
+        $this->subject->setCacheAdapter(new ArrayCacheAdapter);
+
+        $this->assertInstanceOf('OpenGraphParser\ArrayCacheAdapter', $this->subject->getCacheAdapter());
+        $this->assertInstanceOf('OpenGraphParser\ArrayCacheAdapter', $this->subject->getFetchStrategy()->getCacheAdapter());
+    }
+
+    public function testSettingNewFetchStrategyTheCacheAdapterShouldBeAssignedToIt()
```

Comment:
This test is not valid. 

if the setCacheAdapter line is omitted - the test will still pass